### DISCUSSION TITLE
Is Termux Still Good For Learning Kali Hacking?

### DISCUSSION THREAD

**Discussion by B3AST0** [termux/termux-app#2640] [General]
I used this back in 2019 and didn't know much about anything related to linux or hacking but whatever tutorial i followed, it worked. Cut to today when I know my way around Kali (on VMWare on desktop) and know a fair bit of tools. Now there are so many things that just don't work with Termux. I spent so much time trying to run tor networking in vain. There's no aircrack tool. There are so many pentest tools that just don't open for me. When I try to use the gui with nethunter kex, the terminal randomly freezes, none of the sites load in the pre built firefox.

I was just wondering if I am doing something wrong or is Termux not what I thought it to be. Is there any tutorials to completely install Kali on Termux and make it so that all the things that can be done in the cui of VMware (all pentest tools and tor networking)?

**Comment by agnostic-apollo** [comment]
https://wiki.termux.com/wiki/Hacking

**Reply by B3AST0** [reply]
I see....
"If you are interested in maintaining a repository of pentesting or phishing packages, please do it on your own. We encourage community to create own repositories with custom content rather than solely relying on us and our services."
 Would you have any links for packages containing tools like metasploit, aircrack suits, and other pentest tools. I agree with the article and do genuinely want to spend years becoming a hacker/programmer.

**Reply by sk0kanik** [reply]
You also need to search those packages on your own.
From https://wiki.termux.com/wiki/Hacking : "By stepping on hacker's road, especially if decided to use Termux for hacking purposes, you agree that you are on your own. We will not assist you in fixing issues, whether or not related to your activity."

**Reply by B3AST0** [reply]
Huh? And here I was thinking that forums were meant to help each other...
Thnx man I didn't know i have to search the tools on my own 😂

**Reply by sk0kanik** [reply]
If you ask for non-hacking tools, termux community will try to help.

**Reply by B3AST0** [reply]
Ok, is there a community specifically for Termux hacking?

**Reply by thunder-coding** [reply]
> Ok, is there a community specifically for Termux hacking?

There may exist such communities. But we neither support nor encourage you to get engaged in such communities
